grpclb/src/main/java/io/grpc/grpclb/GrpclbConfig.java

```diff
@@ -38,6 +38,10 @@ static GrpclbConfig create(Mode mode) {
     return create(mode, null);
   }
 
+  static GrpclbConfig create(String serviceName) {
```

Comment:
this won't be useful for non test purpose. can you move it to test class?

Reply:
Ok, I thought would be good to have factory method that creates a `GrpclbConfig` with default mode (round_robin). Removed then.